src/components/ApiDelegationOverviewPage/OrgDelegationOverviewPageContent/OrgDelegationOverviewPageContent.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Button,
+  ButtonColor,
+  ButtonSize,
+  ButtonVariant,
+  Panel,
+} from '@altinn/altinn-design-system';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { useAppDispatch, useAppSelector } from '@/rtk/app/hooks';
+import {
+  emptySoftDeletedList,
+  save,
+  setIsEditable,
+  softDeleteAll,
+  softUndeleteAll,
+} from '@/rtk/features/overviewOrg/overviewOrgSlice';
+import { ReactComponent as Add } from '@/assets/Add.svg';
+import { ReactComponent as Edit } from '@/assets/Edit.svg';
+
+import { OrgDelegationAccordion } from './OrgDelegationAccordion';
+import classes from './OrgDelegationOverviewPageContent.module.css';
+
+export const OrgDelegationOverviewPageContent = () => {
+  const overviewOrgs = useAppSelector((state) => state.overviewOrg.overviewOrgs);
+  const softDeletedItems = useAppSelector((state) => state.overviewOrg.softDeletedOverviewOrgs);
+  const isEditable = useAppSelector((state) => state.overviewOrg.overviewOrgIsEditable);
+  const dispatch = useAppDispatch();
+  const [disabled, setDisabled] = useState(true);
+  const { t } = useTranslation('common');
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (softDeletedItems.length >= 1) {
+      setDisabled(false);
+    } else {
+      setDisabled(true);
+    }
+  });
+
+  const accordions = overviewOrgs.map((org) => (
+    <OrgDelegationAccordion
+      key={org.id}
+      organization={org}
+      isEditable={isEditable}
+      softDeleteAllCallback={() => dispatch(softDeleteAll(org.id))}
+      softUndeleteAllCallback={() => dispatch(softUndeleteAll(org.id))}
+    ></OrgDelegationAccordion>
+  ));
+
+  const handleSetIsEditable = () => {
+    if (isEditable) {
+      dispatch(setIsEditable(!isEditable));
```

Comment:
The one in this if has an extra function emptySoftDeletedList()

Reply:
Yes, but if it enters this if, it will perform `dispatch(setIsEditable(!isEditable));` and `dispatch(emptySoftDeletedList());`, and then it will exit the if and perform `dispatch(setIsEditable(!isEditable));` again.

So isn't it enough to have the one setIsEditable that will be performed either way?
(Or is there a reason to setIsEditable being dispatched twice when isEditable is true?)

Reply:
yes it is, will fix it in next push